composer.json

```diff
@@ -18,5 +18,15 @@
         "psr-4": {
             "Calebporzio\\Onboard\\": "src"
         }
+    },
+    "extra": {
+    "laravel": {
+        "providers": [
+            "Calebporzio\\Onboard\\OnboardServiceProvider"
+        ],
+        "aliases": {
+            "Onboard": "Calebporzio\\Onboard\\OnboardFacade"
+        }
     }
+},
```

Comment:
Is that comma required?

Reply:
Oh! No.. It will break the file! Thanks for the hint! :*